robin_stocks/robinhood/authentication.py

```diff
@@ -193,12 +201,49 @@ def login(username=None, password=None, expiresIn=86400, scope='internal', by_sm
                                  'access_token': data['access_token'],
                                  'refresh_token': data['refresh_token'],
                                  'device_token': payload['device_token']}, f)
+        
         else:
-            raise Exception(data['detail'])
+            if 'detail' in data:
+                raise Exception(data['detail'])
+            raise Exception(f"Received an error response {data}")
     else:
         raise Exception('Error: Trouble connecting to robinhood API. Check internet connection.')
     return(data)
 
+def _validate_sherrif_id(device_token:str, workflow_id:str,mfa_code:str):
+    url = "https://api.robinhood.com/pathfinder/user_machine/"
+    payload = {
+        'device_id': device_token,
+        'flow': 'suv',
+        'input':{'workflow_id': workflow_id}
+    }
+    data = request_post(url=url, payload=payload,json=True )
```

Comment:
NIT extra space at end of `request_post` params

Reply:
fixed, I had to disable autoformatting on my end as it was introducing lot of changes on PR.